include/GLFW/glfw3.h

```diff
@@ -1719,6 +1719,31 @@ GLFWAPI GLFWmonitor* glfwGetPrimaryMonitor(void);
  */
 GLFWAPI void glfwGetMonitorPos(GLFWmonitor* monitor, int* xpos, int* ypos);
 
+/*! @brief Returns the work area of the monitor.
+ *
+ *  This function returns the position, in screen coordinates, of the upper-left
+ *  corner of the specified monitor.
```

Comment:
Also size.

Reply:
Added size and work area definition: https://github.com/dougbinks/glfw/commit/1a1ad508abb2d8b626806dc3edc318214e0ff009